code-server/main.tf

```diff
@@ -122,6 +122,16 @@ variable "subdomain" {
   default     = false
 }
 
+variable "open_in" {
+  type        = string
+  description = <<-EOT
+    Determines where the app will be opened. Valid values are `"tab"` and `"slim-window" (default)`.
+    `"tab"` opens in a new tab in the same browser window.
+    `"slim-window"` opens a new browser window without navigation controls.
+  EOT
+  default     = "slim-window"
+}
```

Comment:
Can you add a [`validation`](https://developer.hashicorp.com/terraform/language/values/variables#custom-validation-rules) block to check if the provided value is one of the valid options?

Reply:
done